Commit eaf6486 borked the code.

output of `npm run build:test`:

```
Hash: 93d8e6ee85d61a869706
Version: webpack 3.3.0
Child
    Hash: 93d8e6ee85d61a869706
    Time: 210ms
       [4] ./src/helpers.js 6.6 kB {0} [built] [1 error]
        + 4 hidden modules
    
    ERROR in ./src/helpers.js
    
    /home/david/Repos/tagprobot/src/helpers.js
      200:1  error  Line 200 exceeds the maximum line length of 100  max-len
      201:1  error  Line 201 exceeds the maximum line length of 100  max-len
      202:1  error  Line 202 exceeds the maximum line length of 100  max-len
    
    ✖ 3 problems (3 errors, 0 warnings)
    
     @ ./src/index.js 7:15-35
Hash: 0444e3af0ec33b7916fa
Version: webpack 3.3.0
Child
    Hash: 0444e3af0ec33b7916fa
    Time: 125ms
      [40] ./src/helpers.js 6.04 kB {0} [built]
        + 40 hidden modules
    
    ERROR in failed test: test getTraversableCells - should be equivalent
      Expected: [ [ 1, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 1 ], [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ], [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ], [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ], [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ], [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ], [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ], [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ], [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ], [ 1, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 1 ] ]
      Actual: [ [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ], [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ], [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ], [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ], [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ], [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ], [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ], [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ], [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ], [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ] ]
    
    ERROR in tests failed
```

--- a/src/helpers/map.js
+++ b/src/helpers/map.js
@@ -137,20 +137,19 @@ export function getTraversableCells(cpt, map) {
   }
   for (x = 0; x < xl; x++) {
     for (let y = 0; y < yl; y++) {
-      // TODO: Set radius to be correct value for each cell object. Currently using 29 because it is > 20 * sqrt(2),
-      // the furthest distance from the center of a tile to its corner, in pixels. This guarantees that if the tile
-      // has anything non-tranversable in it (wall, ball, spike, corner wall), the entire tile is marked as non-traversable
+      // TODO: Set radius to be correct value for each cell object. Currently using 29 because it
+      // is > 20 * sqrt(2), the furthest distance from the center of a tile to its corner, in
+      // pixels. This guarantees that if the tile has anything non-tranversable in it (wall, ball,
+      // spike, corner wall), the entire tile is marked as non-traversable
       const objRadius = 29;
       fillGridWithSubgrid(
         emptyCells,
         traversableCellsInTile(
           isTraversable(map[x][y]),
           cpt,
-          objRadius
-        ),
+          objRadius),
         x * cpt,
-        y * cpt
-      );
+        y * cpt);
     }
   }
   return emptyCells;

--- a/src/helpers/map.js
+++ b/src/helpers/map.js
@@ -147,9 +147,11 @@ export function getTraversableCells(cpt, map) {
         traversableCellsInTile(
           isTraversable(map[x][y]),
           cpt,
-          objRadius),
+          objRadius,
+        ),
         x * cpt,
-        y * cpt);
+        y * cpt,
+      );
     }
   }
   return emptyCells;

--- a/tests/map.spec.js
+++ b/tests/map.spec.js
@@ -149,10 +149,10 @@ test('test getTraversableCells', t => {
 
   cpt = 10;
   const smallMap = [[bomb, bluegate]];
-  // For an object with radius 20, the only "traversable" cells
-  // are at the very corners
+  // For an object with radius 29, there are no traversable cells.
+  // TODO: fix this unit test when we have proper object radii
+  // implemented in getTraversableCells
   expected = [
-    [1, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 1],
     [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
     [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
     [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
@@ -161,7 +161,8 @@ test('test getTraversableCells', t => {
     [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
     [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
     [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
-    [1, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 1],
+    [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+    [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
   ];
 
   t.same(map.getTraversableCells(cpt, smallMap), expected);

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -23,9 +23,23 @@ module.exports = function(env) {
             enforce: "pre",
             // Only test js files
             test: [/\.js$/],
-            // Only include files in the client directory (so we don't compile our node modules or server side code)
-            include: TEST_DIR,
+            // Only include files in the client and test directory (so we don't compile our node modules or server side code)
+            include: [TEST_DIR, APP_DIR],
             loader: 'eslint-loader',
+          },
+          // Babel javascript loader, convert js files to es5 javascript
+          {
+            // Only test js files
+            test: [/\.js$/],
+            // Only include files in the src/ directory (so we don't compile our node modules or server side code)
+            include: [TEST_DIR, APP_DIR],
+            loader: 'babel-loader',
+            query: {
+              //use es6 syntax
+              presets: ['es2015'],
+              // makes output more concise
+              plugins: ['transform-runtime'],
+            }
           }
         ]
       },

